python/pyspark/sql/streaming/query.py

```diff
@@ -165,6 +165,11 @@ def explain(self, extended: bool = False) -> None:
 
         Examples
         --------
+        >>> sdf = spark.readStream.format("text").load("python/test_support/sql/streaming")
```

Comment:
I think we can also switch this to `spark.readStream.format("rate").load()`

Reply:
Addressed in https://github.com/apache/spark/commit/7d890d68fc82d860d6783976f30673d11d7ef4c9